src/monda/sorad/qc.py

```diff
@@ -43,7 +44,38 @@ def nearest(items, pivot):
      return nearest_value, nearest_index
 
 
-# Filters used in QC of l and e spectra (Step 1 in 3C and FP chain)
+# Filters used in Step (0) QC (based on tilt and relative azimuth)
+def rel_az_filter(rel_view_az, lower_azi_bound = 110, upper_azi_bound = 170):
+    """ Function to test that So-Rad relative viewing azimiuth is within given angular bounds. 
+    The default lower (110 deg) and upper (170 deg) bounds are based on the AMT cruise defaults"""    
+
+    q_lower =  ~ np.isnan(np.where(np.abs(rel_view_az) < 170, rel_view_az, np.nan))
+    q_upper =  ~ np.isnan(np.where(np.abs(rel_view_az) > 110, rel_view_az, np.nan))
+    q_az =  np.logical_and(q_lower, q_upper)
+    q_az =  np.ravel(q_az.astype(int))
+
+    return q_az
+
+def tilt_filter(tilt_avgs, tilt_stds, upper_tilt_bound=5, upper_tilt_std_bound = np.sqrt(3)):
+    """ Applies upper thresholds to allowed tilt and tilt-standard-deviation values (degs) """
+
+
+    mode_tilt_avgs = mode(tilt_avgs)   # test for constant values  
+    numberstuck = len(tilt_avgs[mode_tilt_avgs == tilt_avgs])
+       
+    if numberstuck > 1:
+        print('Tilt sensor was stuck on single value: do not apply tilt filter')
```

Comment:
I notice we're not using the logger, is that the case just in this script or throughout?

Reply:
We previously only had logger in the test script.  I have imported logger into the qc script, and changed printscreen to log